ansible/roles/nginx/files/genssl.sh

```diff
@@ -56,7 +56,7 @@ else
     openssl req -new \
     -key "$SCRIPTDIR/openwhisk-client-key.pem" \
     -passin pass:$PASSWORD \
-    -subj "/C=US/ST=NY/L=Yorktown/O=OpenWhisk/CN=guest" \
+    -subj "/C=US/ST=NY/L=Yorktown/O=OpenWhisk/CN=whisk.system" \
```

Comment:
* Is guest usable without a cert? I think so but checking.
answer: currently, change from `guest` to `whisk.system`, so there will be no exist client certificate on user `guest`

*  If so then can we avoid changing the catalog scripts?
answer: what's mean for that? i just added `client certificate auth` on `installRouteMgmt.sh`, if client certificate is opened, it will use that. if client certificate is `off`, it will use `auth key` to auth.